docker/Dockerfile-app

```diff
@@ -30,4 +34,4 @@ USER 1001
 
 EXPOSE 8000
 
-CMD ["gunicorn", "--worker-class", "uvicorn.workers.UvicornWorker", "-w 4", "--config", "/src/conf/gunicorn.conf", "ubi_manifest.app.factory:create_app()"]
+CMD ["gunicorn", "ubi_manifest.app.factory:create_app()", "--config", "/src/conf/gunicorn_conf.py"]
```

Comment:
In deployment playbooks we use custom command to run the `app`, I believe it may need also some fix because of configuration passed via different file.

Reply:
I've prepared [draft MR](https://gitlab.cee.redhat.com/exd-guild-ubipop/ubi-manifest-playbooks/-/merge_requests/24) in the playbooks to reflect the change.